samples/mobile/src/main/java/com/gluonhq/maps/samples/mobile/MobileSample.java

```diff
@@ -95,16 +91,7 @@ public void start(Stage stage) throws Exception {
 
         view.flyTo(1., mapPoint, 2.);
     }
-    
-//    private MapLayer myDemoLayer () {
-//        PoiLayer answer = new PoiLayer();
-//        Node icon1 = new Circle(7, Color.BLUE);
-//        answer.addPoint(new MapPoint(50.8458,4.724), icon1);
-//        Node icon2 = new Circle(7, Color.GREEN);
-//        answer.addPoint(new MapPoint(37.396256,-121.953847), icon2);
-//        return answer;
-//    }
-    
+
     private MapLayer positionLayer() {
         return Services.get(PositionService.class)
                 .map(positionService -> {
```

Comment:
The new PositionService requires this to get started:
`positionService.start();`